hw7_FP/functional.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+def sequential_map(*args):
+    result=map(args[0], args[-1])
+    for func in args[1:-1]:
+        result=map(func,list(result))   
+    return list(result)
```

Comment:
1. Пробелы!!!
2. А зачем как-то отдельно вне цикла применять первую функцию, в цикле же можно сделать сразу всё
3. Превратить элемент в список можно только в самом конце, map можно передавать в другие map'ы
```suggestion
    *functions, container = args
    for function in functions:
        container = map(function, container)   
    return list(result)
```